ci/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nvidia/cuda:8.0-devel
 
-ENV N_PROCS=4
+ENV N_PROCS=1
```

Comment:
Why did you change the default?

Reply:
Because I can't compile with 4 procs on my desktop. deal uses all the memory.